api/src/extensions.ts

```diff
@@ -488,6 +498,19 @@ class ExtensionManager {
 					logger.warn(`Couldn't register cron hook. Provided cron is invalid: ${cron}`);
 				}
 			},
+			embed: (position: 'head' | 'body', code: string | EmbedHandler) => {
+				const content = typeof code === 'function' ? code(env) : code;
+				if (content.trim().length === 0) {
+					logger.warn(`Couldn't register embed hook. Provided code is empty!`);
+					return;
+				}
+				if (position === 'head') {
+					this.hookEmbedsHead.push(content);
+				}
+				if (position === 'body') {
+					this.hookEmbedsHead.push(content);
```

Comment:
Shouldn't this be `this.hookEmbedsBody.push(content)` ?

Reply:
It definitely should, thanks 😄